scripts/abundance-dist.py

```diff
@@ -93,7 +97,14 @@ def main():
         print("\tPlease verify that the input files are valid.",
               file=sys.stderr)
         sys.exit(1)
+
     hash_fp = open(args.output_histogram_filename, 'w')
+    if args.csv:
+        hash_fp_csv = csv.writer(hash_fp)
+        # write headers:
+        hash_fp_csv.writerow(['k-mer_abundance', 'K-mer_count',
```

Comment:
please makes these lowercase, no dashes, and short; I suggest abundance, count, cumulative, cumulative_fraction, so that they fit better as variable names.